### PR TITLE
Update documentation for signup()

### LINKED ISSUE #151: On signup(), make it clear you are not logged in
Right now, if you call signup(), you will not be logged in, you still have to call login(). This has to be made clear in the documentation, or we should make it automatic to be logged in after doing signup().

### PR DESCRIPTION
**Pull request by ussgarci**
# Description

```
Right now, if you call signup(), you will not be logged in, you still have to call login(). This has to be made clear in the documentation.
```
See discussion on issue #151 for more details.

Fixes #151 

## Type of change

Please select the option(s) that is more relevant.

- [ ] Code cleanup
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### CODE REVIEW

File: web/docs/language/features.md [hunk 1]
```diff
@@ -839,6 +839,7 @@ The quickest way to get started is by using the following API generated by Wasp:
 - Signup and Login forms at `@wasp/auth/forms/Signup` and `@wasp/auth/forms/Login` routes
 - `logout` function
 - `useAuth()` React hook
+It should be noted that the Signup form automatically logs in the user after signup.
```
Comment (sodic) [inline web/docs/language/features.md:842]:
```suggestion
**NOTE:** If the signup is successful, the Signup form will automatically log in the user.
```
Just to follow the style we use in the rest of the docs :)

File: web/docs/language/features.md [hunk 2]
```diff
@@ -909,7 +910,7 @@ Login is a regular action and can be used directly from the frontend.
 
 
 ### `signup()`
-An action for signing in in the user.
+An action for signing up the user. This action does not login a user, you still need to call `login()`.
```
[the remark this answers is not in the document]
Comment (ussgarci) [reply web/docs/language/features.md]: Learned something new today.  Thanks!

File: web/docs/language/features.md (hunk 1 above)
Comment (ussgarci) [reply web/docs/language/features.md]: Excellent.  :+1:

Comment (ussgarci) [reply web/docs/language/features.md]: Wow, I just saw that I can commit the suggestion from the UI.  :exploding_head:

Comment (sodic) [reply web/docs/language/features.md]: Yeah, pretty cool. I've only learned about it recently too.

**Pull request merged**